volumetric_imaging/anaconda-project.yml

```diff
@@ -0,0 +1,51 @@
+name: volumetric_imaging
+description: Display 3D image and segmentation data in a Jupyter notebook.
+
+examples_config:
+  created: 2024-11-22
+  maintainers:
+  - "droumis"
+  labels:
+  - "panel"
+  categories:
+  - Neuroscience
+  title: "Volumetric Imaging"
+  deployments:
+  - command: notebook
+  # - command: dashboard
+  no_data_ingestion: true
+
+user_fields: [examples_config]
+
+channels:
+- conda-forge
+- nodefaults
+
+packages: &pkgs
+- notebook>=6.5.2,<7 # required
+- python=3.11
+- panel
+- jupyterlab
+- pip
+- pip:
+  - neuroglancer
+
+dependencies: *pkgs
+
+commands:
+  lab:
+    unix: jupyter lab volumetric_imaging.ipynb
+    windows: jupyter lab volumetric_imaging.ipynb
+  notebook:
+    notebook: volumetric_imaging.ipynb
+  dashboard:
```

Comment:
There's no `.servable()` in the notebook. Unless you want to deploy it just like that with this newish feature of Panel? That would be a first here!

<img width="855" alt="image" src="https://github.com/user-attachments/assets/25d28c23-c3d9-4f7a-9688-1a5b127cf8f2">

Reply:
It doesn't really make sense as a Panel app in its current form. 

I suggest we add a colab command for notebook use.

Additionally, I'm ok with leaving this newish deploy app thing for panel (does it have a name?).